a-few-things-you-should-know-about-javascript.md

```diff
@@ -203,12 +203,12 @@ For end-to-end testing I use Nightmare. I try to keep my number of end-to-end te
 
 ## Final Thoughts
 
-JavaScript offers some unique benefits and makes a great candidate for building applications on any platform. Due to the amount of places it can run, and it’s ability to be written in ways that make code extremely reusable and composable, JavaScript is probably the most important language you can know right now.
+JavaScript offers some unique benefits and makes a great candidate for building applications on any platform. JavaScript is probably the most important language you can know right now, because it can be run virtually anywhere and can be written in highly reusable, composable ways.
 
-However, those same benefits in the hands of those unfamiliar with the differences and “gotchyas” of JavaScript, may quickly find that making changes to the codebase is not as easy as it once was and changes seem to have adverse and unexpected results.
+However, those unfamiliar with the differences and “gotchas” of JavaScript may quickly find that making changes to the codebase is not as easy as it once was and changes seem to have adverse and unexpected results.
 
 Don’t get caught caught up in writing applications like you do for other languages. Make the most of what JavaScript has to offer and work towards small and simple modules. This will help keep you sane and your love for JavaScript strong!
 
-I would love to know how others achieve velocity and keep JavaScript heavy environments running smoothly, please share your processes and modules of choice in the comments below!
+I would love to know how others achieve velocity and keep JavaScript heavy environments running smoothly. Please share your processes and modules of choice in the comments below!
```

Comment:
actually it's better if you say "I would love to know how **you** achieve velocity" because it sounds more personal.